src/utils/GetTestResults.js

```diff
@@ -28,6 +28,22 @@ class GetTestResults {
       return dateFns.isAfter(testResult.testStartTimestamp, fromDateTime) && dateFns.isBefore(testResult.testEndTimestamp, toDateTime)
     })
   }
+
+  static filterTestResultsByDeleteFlag (testResults) {
+    return testResults.filter((testResult) => {
+      return !testResult.deleteFlag === true
+    })
+  }
+
+  static filterTestTypesByDeleteFlag (testResults) {
+    testResults.forEach(testResult => {
```

Comment:
I'd normally choose to use map over forEach, for speed and immutability, but it probably doesn't matter much here. The only difference is that map returns a new array with your changed values, rather than altering the original, so you'd want to put the return on the top.